ethereum-cpp-on-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,224 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "newStorageAccountName": {
```

Comment:
Can you take a look at our new guidance for storage account names and update? https://github.com/Azure/azure-quickstart-templates/issues/944